### ISSUE TITLE
Request: Speed reading option for teleprompter

### ISSUE THREAD

**Issue by bigsmile01** [Nosrac/Dictater#17]
It would be neat if teleprompter would only show one word on screen at a time adjusting the size of the word to fit the teleprompter window. 

This would be useful for:
-Teleprompter reading: As you only see the word you have to speak.
-Proofing documents: I use dictate for proofreading, as it is good way to spot misspelt words (thanks to the audio) and missing punctuation (as the teleprompter highlights word). Having one word on screen at a time would make proofreading even easier (especially for spotting mixed up words).

**Comment by amakukha** [comment]
Agree. I'd like to use it for speed-reading. For example, when I just need to go through some boring text, but don't have the time to read it thoroughly.

With "say" utility on the terminal I can set the speech rate to 300 with option "-r". But that utility doesn't have the pause button when I want to read through some part myself.

So I wish, Dictater had the reading speed option.
